front/extension/app/src/components/auth/ProtectedRoute.tsx

```diff
@@ -48,7 +67,11 @@ export const ProtectedRoute = ({ children }: { children: ReactNode }) => {
           size="sm"
         />
       </div>
-      <>{children}</>
+      <>
+        {typeof children === "function"
+          ? children({ user, workspace })
+          : children}
+      </>
```

Comment:
Can we put workspace/user in a context instead ? It would be easier to access in sub components

Reply:
Very fair

Reply:
Discussed IRl we're going with that and will improve maybe later